src/main/index.js

```diff
@@ -5,9 +5,12 @@ import configureMenu, { dockMenu } from './menus'
 import createWindow, { windows } from './createWindow'
 // import { autoUpdater } from "electron-updater"
 
-const openFilesCache = []
+var openFilesCache = []
```

Comment:
You should use `let` instead of `var` to allow scope isolation. More info : https://medium.com/craft-academy/javascript-variables-should-you-use-let-var-or-const-394f7645c88f

Reply:
Woops, @Jocs was faster 😆

Reply:
😃

Reply:
I think next time I will use some code analysis tool.